### PR TITLE
issue: 2583763 Set limit of threads

### PR DESCRIPTION
**Pull request by AoD314**
## Description
Set limit of threads to stable CI

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

File: contrib/jenkins_tests/cov.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ make clean >> "${cov_dir}/cov.log" 2>&1
 sleep 1
 eval "cov-configure --config ${cov_dir}/coverity_config.xml --gcc >> ${cov_dir}/cov.log 2>&1"
 sleep 1
-eval "cov-build --config ${cov_dir}/coverity_config.xml --dir ${cov_build} make -j >> ${cov_dir}/cov.log 2>&1"
+eval "cov-build --config ${cov_dir}/coverity_config.xml --dir ${cov_build} make -j8 >> ${cov_dir}/cov.log 2>&1"
```
Comment (igor-ivanov) [inline contrib/jenkins_tests/cov.sh:27]: please consider using `make ${make_opt}`

File: contrib/jenkins_tests/rpm.sh [hunk 2]
```diff
@@ -32,7 +32,8 @@ else
                --define='_sourcedir ${rpm_dir}' \
                --define='_specdir ${rpm_dir}' \
                --define='_builddir ${rpm_dir}' \
-               --define='_tmppath ${rpm_dir}/_tmp'"
+               --define='_tmppath ${rpm_dir}/_tmp' \
+               --define='%_smp_mflags -j8'"
```
Comment (igor-ivanov) [inline contrib/jenkins_tests/rpm.sh:36]: please consider using make ${make_opt}

File: contrib/jenkins_tests/rpm.sh [hunk 3]
```diff
@@ -75,7 +76,7 @@ if [ $opt_binrpm -eq 1 ]; then
     if [ $opt_rpm -eq 1 ]; then
         test_exec="rpmbuild -bb $rpmmacros $rpmopts $rpmspec"
     else
-        test_exec="dpkg-buildpackage -us -uc -b"
+        test_exec="dpkg-buildpackage -us -uc -b -j8"
```
Comment (igor-ivanov) [inline contrib/jenkins_tests/rpm.sh:79]: please consider using ${make_opt} instead of -j8

File: .ci/matrix_job.yaml [hunk 4]
```diff
@@ -34,7 +37,7 @@ runs_on_dockers:
 # inbox
   - {name: 'ub21.10-inbox-x86_64',  url: 'harbor.mellanox.com/hpcx/x86_64/ubuntu21.10/builder:inbox', category: 'base', arch: 'x86_64'}
   - {name: 'rhel8.3-inbox-x86_64',  url: 'harbor.mellanox.com/hpcx/x86_64/rhel8.3/builder:inbox', category: 'base', arch: 'x86_64'}
-  - {name: 'rhel8.3-inbox-aarch64', url: 'harbor.mellanox.com/hpcx/aarch64/rhel8.3/builder:inbox', category: 'base', arch: 'aarch64'}
+  # - {name: 'rhel8.3-inbox-aarch64', url: 'harbor.mellanox.com/hpcx/aarch64/rhel8.3/builder:inbox', category: 'base', arch: 'aarch64'}
```
Comment (igor-ivanov) [inline .ci/matrix_job.yaml:40]: please suggest an alternative with aarch64

File: contrib/jenkins_tests/cov.sh (hunk 1 above)
Comment (AoD314) [reply contrib/jenkins_tests/cov.sh]: fixed

File: contrib/jenkins_tests/rpm.sh (hunk 2 above)
Comment (AoD314) [reply contrib/jenkins_tests/rpm.sh]: fixed

File: contrib/jenkins_tests/rpm.sh (hunk 3 above)
Comment (AoD314) [reply contrib/jenkins_tests/rpm.sh]: fixed

File: .ci/matrix_job.yaml (hunk 4 above)
Comment (AoD314) [reply .ci/matrix_job.yaml]: fixed

File: contrib/jenkins_tests/cov.sh (hunk 1 above)
Comment (lennybe) [reply contrib/jenkins_tests/cov.sh]: @igor-ivanov , please review and approve

**Review by lennybe — APPROVED** [review]

File: .ci/matrix_job.yaml [hunk 5]
```diff
@@ -442,3 +445,5 @@ pipeline_stop:
 failFast: false
 
 taskName: '${flags}/${name}/${axis_index}'
+
+timeout_minutes: 120
```
Comment (AlexanderGrissik) [inline .ci/matrix_job.yaml:449]: Is this used somewhere?

**Review by AlexanderGrissik — APPROVED** [review]

**Pull request merged**